scripts/build.d/qt

```diff
@@ -4,33 +4,49 @@ set -e
 
 if [ -z "${SYNCGIT}" ]; then
   pkgname=qt
-  pkgmajorver=${MAJOR_VER:-6.4}
-  pkgsubver=${SUB_VER:-1}
+  pkgmajorver=${MAJOR_VER:-6.5}
+  pkgsubver=${SUB_VER:-0}
   pkgver=$pkgmajorver.$pkgsubver
-  pkgmd5=${MD5:-ae18c8d4c2d0b8fb757c6881f6e273ea}
+  pkgmd5=${MD5:-10247444e4264ea9cee7d4a7c13efd34}
   pkgfull=$pkgname-$pkgver
   pkgfoler=qt-everywhere-src-$pkgver
 
   pkgfn=$pkgfull.tar.xz
   pkgurl=https://download.qt.io/official_releases/qt/$pkgmajorver/$pkgver/single/qt-everywhere-src-$pkgver.tar.xz
 
+  pkgsrc=${DEVENVFLAVORROOT}/${DEVENVFLAVOR}/src
+  #pkgsrc=/tmp
+
   download_md5 $pkgfn $pkgurl $pkgmd5
 
-  mkdir -p ${DEVENVFLAVORROOT}/${DEVENVFLAVOR}/src
-  pushd ${DEVENVFLAVORROOT}/${DEVENVFLAVOR}/src > /dev/null
-    tar Jxf ${DEVENVDLROOT}/$pkgfn
+  mkdir -p $pkgsrc
+  if [ -d $pkgsrc/$pkgfull ];
```

Comment:
I missed this.  It behaves differently from other build scripts.  Please add an environment variable to control the behavior.

Reply:
For example, https://github.com/solvcon/devenv/blob/8e7898bf43a379cb672ec9fd7d061119b3fe8ad3/scripts/build.d/python#L16